### DISCUSSION TITLE
Unable to upload photo to a private github project

### DISCUSSION THREAD

**Discussion by paulperry** [community/community#47026] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I was unable to upload a png photo to an organization level private github project.  I immediately get the following:

I then tried to upload this very input window and I get a proper error message stating the png > 10MB.  Changing the image size allowed me to proceed.  A proper error message, otherwise other team members would have me switch to some other system :/ .

**Comment by Hangell** [comment]
O GitHub tem restrições de tamanho de arquivo para imagens e outros tipos de arquivos para manter a eficiência do sistema. O limite de tamanho de imagem é de 10 MB por arquivo.

**Comment by KsenijaGogic** [comment]
I seem to be unable to upload at all using Chrome (110.0.5481.77). The error that shows is:
"We don't support that file type. Try again with a GIF, JPEG, JPG, MOV, MP4, PNG, SVG, WEBM, CSV, DOCX, FODG, FODP, FODS, FODT, GZ, LOG, MD, ODF, ODG, ODP, ODS, ODT, PATCH, PDF, PPTX, TGZ, TXT, XLS, XLSX or ZIP."

I tried uploading: PNG (original) and JPEG. Both fail.

I am, however, able to upload via Firefox (109.0.1) without issue.
